package.json

```diff
@@ -89,11 +89,11 @@
     "deepmerge": "^4.3.1",
     "eventsource": "^2.0.2",
     "fast-json-stringify": "^6.0.0",
-    "fastify": "^4.28.1",
-    "fastify-plugin": "^4.5.1",
-    "fastify-raw-body": "^4.3.0",
+    "fastify": "^5.0.0",
```

Comment:
It turns out v5 has some breaking changes, have to go through migration guide https://fastify.dev/docs/latest/Guides/Migration-Guide-V5/ - does not look as that large of a change